e2e/support/helpers/e2e-qa-databases-helpers.js

```diff
@@ -214,6 +214,14 @@ export function queryWritableDB(query, type = "postgres") {
   });
 }
 
+export function resetWritableDb({ type = "postgres" }) {
+  cy.log(`Resetting ${type} writable DB`);
+  cy.task("resetWritableDb", { type });
+}
+
+/**
+ * Note: this function MUST come after the restore() function in the file, or it will get wiped out
+ */
```

Comment:
this seems like a super-easy mistake to make, there probably should be some enforcing mechanism for this, like an eslint rule or something. otherwise I think we’ll confuse a lot of people

Reply:
[DEVX-122: Write lint rule to prevent writable db bugs](https://linear.app/metabase-inc/issue/DEVX-122/write-lint-rule-to-prevent-writable-db-bugs)